### ISSUE TITLE
Account for WGD

### ISSUE THREAD

**Issue by teng-gao** [kharchenkolab/numbat#36]
Ideally, we should account for WGD (baseline is 4 copies) when analyzing hyperploid tumors (e.g. TNBC5), which should lead to better model fit and prediction stability

**Comment by DarioS** [comment]
Isn't the baseline usually at three copies because of fitness? I have a cancer with 70% of samples with whole genome duplication and genome ploidy from PURPLE is rarely near four but often close to three, so I am looking forward to this enhancement!

![image](https://github.com/kharchenkolab/numbat/assets/631218/8a3834b0-e2ce-4d1c-893d-8aa018cbbdb4)

> Whereas chromosome losses are rarely tolerated in diploid cells, they occur frequently in tetraploid cells and can promote cancer formation.

**Comment by teng-gao** [comment]
> Isn't the baseline usually at three copies because of fitness? I have a cancer with 70% of samples with whole genome duplication and genome ploidy from PURPLE is rarely near four but often close to three, so I am looking forward to this enhancement!

Right, although tumors that have ploidy near 3 should be ok unless no diploid regions are available anywhere on the genome. The issue only arises when the lowest (in copy number) balanced segments are 4 copies.

**Comment by DarioS** [comment]
I noticed that non-WGD samples have near-perfect agreement to whole genome sequencing (PURPLE) but not WGD.
![image](https://github.com/kharchenkolab/numbat/assets/631218/15f225c2-d2c5-4221-bf8f-c776ef312b63)
It seems implausible that all of a sample's arms would be lost or gained rather than a mix. Sample IDs ending in Bulk are WGS.
